code/docker-compose.dev.yaml

```diff
@@ -11,7 +11,7 @@ services:
       ara-dev: {}
     environment:
       - INFO_APP_VERSION=DEV
-      - ARA_OAUTH2_MODE=oauth2-mock
+      #- ARA_OAUTH2_MODE=oauth2
```

Comment:
do not comment: if not used, simply remove it

Reply:
to be able to use oauth2 custom